lib/params/FlowParams.cpp

```diff
@@ -206,11 +206,21 @@ std::vector<float> FlowParams::GetRake() const
     auto doubles = GetValueDoubleVec( _rakeTag );
     auto rakesize = doubles.size();
 
+    // If the rakesize isn't correct, then it isn't initialized yet.
+    // Let's use the values stored in _Box to initialize the rake.
     if ( rakesize != 6 && rakesize != 4 ) 
     {
-        // Six NANs represent the initial state
-        std::vector<float> tmp(6, std::nan("1") );
-        return tmp;
+        std::vector<double> minext, maxext;
```

Comment:
Rather than initializing this in the getter, why not do the initialization in the constructor as is convention? It would also be consistent with the way Box class is initialized in the Renderer class.

Reply:
Hi John, the rake initialization has been moved to the constructor in commit 86a4725, and this branch is ready to be reviewed again.